source/styles/typography.less

```diff
@@ -1,12 +1,11 @@
 // Loading custom fonts
 @font-face {
   font-family: "Akkurat-Light";
-  src:url("../../applications/fonts/lineto-akkurat-light.eot");
-  src:url("../../applications/fonts/lineto-akkurat-light.eot?#iefix") format("embedded-opentype"),
-  url("../../applications/fonts/lineto-akkurat-light.woff2") format("woff2"),
-  url("../../applications/fonts/lineto-akkurat-light.woff") format("woff");
+  src: url("../../applications/fonts/lineto-akkurat-light.woff2") format("woff2"),
+  url("../fonts/lineto-akkurat-light.eot?#iefix") format("embedded-opentype"),
+  url("../fonts/lineto-akkurat-light.eot");
   font-weight: normal;
-  font-style: normal;
+  font-style: normal
```

Comment:
Hi Ellie, please can you add back in the ; ?  Thanks.

Reply:
Done 👍